flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/mysql/MysqlType.java

```diff
@@ -92,10 +92,22 @@ public class MysqlType {
     private static final String JSON = "JSON";
     private static final String ENUM = "ENUM";
     private static final String SET = "SET";
+    private static final String GEOMETRY = "GEOMETRY";
+    private static final String POINT = "POINT";
+    private static final String LINESTRING = "LINESTRING";
+    private static final String POLYGON = "POLYGON";
+    private static final String MULTIPOINT = "MULTIPOINT";
+    private static final String MULTILINESTRING = "MULTILINESTRING";
+    private static final String MULTIPOLYGON = "MULTIPOLYGON";
+    private static final String GEOMETRYCOLLECTION = "GEOMETRYCOLLECTION";
 
     public static String toDorisType(String type, Integer length, Integer scale) {
         switch (type.toUpperCase()) {
             case BIT:
+                if (length > 1) {
+                    return DorisType.STRING;
+                }
+                return DorisType.BOOLEAN;
```

Comment:
What is the purpose of adding this judgment here?

Reply:
when the length of bit is greater than 1,it will convert to a string.